src/scripts.js

```diff
@@ -110,13 +185,13 @@ function displayRecipePreview(current, view) {
     <div class = "fullwrap" id="${current.id}">
     <span id="favorite">❤️</span>
```

Comment:
We got rid of the heart to avoid code conflict and most of our addRecipeToFavorites is different now, stay tuned for the changes